presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/DesugarCurrentUser.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner.iterative.rule;
+
+public class DesugarCurrentUser
+        extends ExpressionRewriteRuleSet
+{
+    public DesugarCurrentUser()
+    {
+        super(createRewrite());
+    }
+
+    private static ExpressionRewriter createRewrite()
+    {
+        return (expression, context) -> DesugarCurrentUserRewriter.rewrite(expression);
```

Comment:
Inline the rewrite method. There's no point in having them separate. (discussed with @martint and he agrees)